boards/arm/nrf9160_pca20035/adp536x/adp536x.c

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
```

Comment:
I think this file shouldn't be in `/boards`, perhaps we can we find a better place under `/drivers` ?